tests/core/test_entities.py

```diff
@@ -32,22 +28,21 @@
 MONTH = "2016-01"
 
 
-def new_simulation(test_case, period = MONTH):
-    builder = SimulationBuilder()
-    builder.set_default_period(period)
-    return builder.build_from_entities(tax_benefit_system, test_case)
+def new_simulation(simulation_builder, tax_benefit_system, test_case, period = MONTH):
```

Comment:
Related to my comment in `tests/fixtures/simulations.py` ... 
Why use the 'simulation_builder' fixture here instead of instantiating SimulationBuilder() directly?

The existing code is more readable IMHO.

Reply:
Changed.